pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -232,6 +248,23 @@ def newpkgcb(pkgId, name, arch):
             result = await downloader.run()
             metadata_pb.increment()
 
+            kickstart_parsed = {}
+            namspaces = [".treeinfo", "treeinfo"]
+            for namespace in namspaces:
+                response = requests.get(urljoin(self.remote.url, namespace))
+                if response.status_code == 200:
```

Comment:
This look like it's actually checking a response status code from `requests`.

Reply:
yep, it is for checking which URL exists, the one which ends with `.treeinfo` or `treeinfo`